src/Store.ts

```diff
@@ -1,4 +1,4 @@
-import { Store, Reducer, createStore, combineReducers } from 'redux';
+import { Store, Reducer, createStore as reduxCreateStore, combineReducers } from 'redux';
 import { Activity, IBotConnection, User } from './BotConnection';
 import { FormatOptions } from './Chat';
 import { strings, Strings } from './Strings';
```

Comment:
I propose exporting type ChatStore = Store<ChatState> - that way all the places where you set props: Store<ChatStore> can become props: ChatStore, which is one fewer import and a simpler declaration